app/3.0/start/toolbox/set-up.md

```diff
@@ -35,9 +35,10 @@ To install Polymer CLI:
 
         polymer init
 
-    Press the down arrow until `polymer-2-starter-kit` is highlighted and press the enter / return
+    Press the down arrow until `polymer-3-starter-kit` is highlighted and press the enter / return
     key to select.
 
+Polymer CLI downloads and installs dependencies, and creates an app from the `polymer-3-start-kit template`.
 
 ## Serve your project
 
```

Comment:
I think we might want to change this section to show `polymer serve` without `--open` and mention that the user should go to the url shown after `applications:`. `polymer serve --open` opens the component-style URL which doesn't work anymore.